custom_components/vivint/switch.py

```diff
@@ -30,6 +31,13 @@ def has_capability(device: VivintDevice, category: Category, capability: Capabil
     return False
 
 
+def has_feature(device: VivintDevice, feature: Feature):
+    """Check if a device has a feature."""
+    if feature in (device.features or {}):
```

Comment:
It's probably more syntactically correct to use `device.features or []` since features is a list now. But {} can also be an empty set so the check will still work

Reply:
Also, you should probably just do `return feature in (device.features or [])`